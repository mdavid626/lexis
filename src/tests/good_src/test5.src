/' Program 5: Jednoduchy print 2'/

Scope 'Hlavni telo programu
  Dim a as Integer
  Dim b as Double
  dim s as string
  a = 24
  b = 2.3
  s = "hello \nWorld\\abc"
  print a ; b ; s ;
End Scope
