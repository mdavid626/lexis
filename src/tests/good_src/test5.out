 24 2.3hello 
World\abc